src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Source/SourceUpdaterStep.cs

```diff
@@ -116,7 +116,8 @@ public async Task GetDiagnosticsAsync(CancellationToken token)
             Logger.LogTrace("Running ASP.NET Core upgrade analyzers on {ProjectName}", project.Name);
 
             // Compile with upgrade analyzers enabled
-            var applicableAnalyzers = GetApplicableAnalyzers(_allAnalyzers, Project);
+            var applicableAnalyzers = await GetApplicableAnalyzers(_allAnalyzers, Project).ToListAsync(token).ConfigureAwait(false);
```

Comment:
AsyncEnumerable should be a method named async

Reply:
The pattern for async enumerables is to not include the async.